moped-editor/src/utils/urls.js

```diff
@@ -0,0 +1,11 @@
+import * as yup from "yup";
+
+/**
+ * Validate a url using yup
+ * @see https://github.com/jquense/yup/blob/1086aa93fdd08a554936a409c30788058dbc7c32/src/string.ts#L24-L26
```

Comment:
thank you for including the repo link/the line to the specific logic in this comment, it makes it super easy to go and checkout how yup works!